src/EnergyPlus/PluginManager.cc

```diff
@@ -705,7 +705,7 @@ namespace PluginManagement {
 
             for (unsigned long itemNum = 0; itemNum < numVals; itemNum++) {
                 PyObject *item = PyList_GetItem(pyth_val, itemNum);
-                if PyUnicode_Check(item) { // NOLINT(hicpp-signed-bitwise) -- something inside Python code causes warning
+                if (PyUnicode_Check(item)) { // NOLINT(hicpp-signed-bitwise) -- something inside Python code causes warning
```

Comment:
I do not understand how this was compiling before.  The macro expansion does not result in a parenthesized expression.  

Thanks for catching this!

Reply:
@mjwitte Did you upgrade to python 3.9 by any chance?

I'm tracking the current cpython develop versus 3.8.6 and I think I know why.

Here is current develop of cpython, (I made adjustements to avoid python types): https://godbolt.org/z/ncYWzj
https://github.com/python/cpython/blob/5977a7989d49c3e095c7659a58267d87a17b12b1/Include/object.h#L672-L686


https://godbolt.org/z/Prrhao

https://github.com/python/cpython/blob/db455296be5f792b8c12b7cd7f3962b52e4f44ee/Include/object.h#L348-L351

Reply:
@jmarrec I did, actually install 3.9. Not by choice, but because I had to reinstall everything  due to a system failure.

Reply:
Thanks Python :laughing: